app/mailers/dependents_application_failure_mailer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class DependentsApplicationFailureMailer < ApplicationMailer
+  def build(user_hash)
```

Comment:
Might be a good idea to validate the input at the beginning of `def build(user_hash)` just in case `@user_hash['first_name']`, `@user_hash['last_name']` or `@user_hash['email']` are `nil`. Something like :
```ruby
def build(user_hash)
  user = User.new(user_hash)
  raise "Bad user #{user.errors}" unless user.valid?
```

Probably nitpicky, but my 2 cents.

Reply:
@omgitsbillryan I think I'd rather the veteran get the email without the personalization than raise and them not get it. WDYT? 🤔